src/main/java/org/fcrepo/audit/InternalAuditor.java

```diff
@@ -190,7 +186,6 @@ public void releaseConnections() {
      */
     public void createAuditNode(final FedoraEvent event) throws RepositoryRuntimeException, IOException {
```

Comment:
Is there any reason that updates to `InternalAuditorTest` are not included in this PR?